contrib/cmd/seccompagent/seccompagent.go

```diff
@@ -0,0 +1,193 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"net"
+	"os"
+
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"golang.org/x/sys/unix"
+)
+
+/*
+#include <inttypes.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/ioctl.h>
+
+// Flags for seccomp notification fd ioctl.
```

Comment:
Can we do something like

```c
#ifndef SECCOMP_IOCTL_NOTIF_RECV
#define SECCOMP_IOCTL_NOTIF_RECV	SECCOMP_IOWR(0, struct seccomp_notif)
#endif

/* etc */
```

Reply:
Mostly, because of things like: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=47e33c05f9f07cac3de833e531bcac9ae052c7ca

Reply:
Mostly, because of things like: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=47e33c05f9f07cac3de833e531bcac9ae052c7ca